### DISCUSSION TITLE
Download user data from Jacquard tag

### DISCUSSION THREAD

**Discussion by msuardi** [google/JacquardSDKAndroid#4] [General]
I have an Adidas GMR Jacquard tag and I would like to use the Jacquard SDK in order to connect the tag to the app and download the user data recorded by the sensor (raw data or the output of the models inside the tag).
I have successfully run the sample app, so I'm able to connect to the tag.
How can I download the user data now?
Thank you!

**Comment by wsehringer** [comment]
Hi @msuardi , you were able to solve this issue?, the same thing happens to me
